Application/EdFi.Admin.DataAccess/Contexts/PostgresUsersContext.cs

```diff
@@ -14,22 +14,18 @@ namespace EdFi.Admin.DataAccess.Contexts
     public class PostgresUsersContext : UsersContext
     {
         public PostgresUsersContext(DbContextOptions options) : base(options) { }
-        
-        protected override void ApplyProviderSpecificMappings(ModelBuilder modelBuilder)
-        {
-            modelBuilder.Entity<ApiClient>()
-                .HasMany(t => t.ApplicationEducationOrganizations)
-                .WithMany(t => t.Clients)
-                .UsingEntity(join => join.ToTable("apiclientapplicationeducationorganizations"));
-
-            modelBuilder.UseUnderscoredFkColumnNames();
 
-            modelBuilder.Model.FindEntityTypes(typeof(ApiClient)).First().GetProperty("CreatorOwnershipTokenId")
-                .SetColumnName("creatorownershiptokenid_ownershiptokenid");
```

Comment:
The prior code was not quite working, especially with respect to overriding the column name `applicationedorg_applicationedorgid`. I was able to adopt the code from the `main` branch without breaking anything else, and it seems to be working.